news/2021-11-09-performance-points-star-rating-updates.md

```diff
@@ -12,6 +12,49 @@ If you find yourself scratching your head when reading, consider consulting the
 
 We will be covering only changes made to the **osu!** game mode in this post.
 
+## Release Schedule
+
+Deployment of these changes will as usual, take some time to fully complete. As this batch of changes affects all game modes, it will likely take longer than normal. If you are interested in following the progress we recommend you bookmark this page and check back on this section, as it will be updated as forward progress is made.
+
+### 🏃 Star rating updates \[2021-11-09 - 2021-11-11\]
+
+An initial pass over all beatmaps to update their star ratings. Usually we wouldn't pause pp updates during this stage, but due to new additions it is required this time.
```

Comment:
what do you mean by "pause pp updates during this stage" ? does the score submission will not be possible during the stage ? maybe more clarification about that.

Reply:
Pretty sure this refers to freezing rank history graphs

Reply:
This means freezing *all* updates to the point I'm not sure we will ever catch up.

Reply:
> Pretty sure this refers to freezing rank history graphs

People will understand this better than pause pp updates, I guess

Reply:
Pending further consideration. I don't think it's going to work as we have it.

Reply:
It has nothing to do with rank graph updates.

Reply:
so it do not pause the pp update because it does not start at this point if im not wrong. You can say that it start by updating the star rating and after that the pp update begin. Is that what you mean ?

Reply:
In my opinion saying that freezing updates to pp makes a lot more sense.
Just in case add a extra note what it means as people may get confused.

Reply:
I think having a note, reassuring that this won't affect score submission would probably good, cause that's always peoples biggest fear.

Reply:
I understand it now. If you add that it contains all pp updates, people might get less confused.

Reply:
If I understood it correctly, it means that when you submit a play, your pp won't update because the updates are currently paused due to the rework process.

If I am correct on that, the note should say something like: "Any plays you make during that time window will be submitted, but any pp values won't be updated."